WikiDrips/WikiTableViewController.swift

```diff
@@ -179,16 +196,16 @@ extension WikiTableViewController: UITableViewDataSourcePrefetching {
         }
     }
     
-// No need for cancelPrefetchingForRowsAt since changing search cancels pending operations and prefetchRowsAt populates a cache
+    // No need for cancelPrefetchingForRowsAt since changing search cancels pending operations and prefetchRowsAt populates a cache
```

Comment:
I hadn't seen this before. I think you do need `cancelPrefetchingForRowsAt` when you're scrolling a long list quickly then I think you'll get this method called when on rows that the table view no longer needs.